source/module_esolver/esolver_ks.cpp

```diff
@@ -16,6 +16,7 @@
 #include "module_base/global_variable.h"
 #include "module_hamilt_lcao/module_dftu/dftu.h"
 #include "module_hamilt_pw/hamilt_pwdft/global.h"
+#include "module_base/parallel_reduce.h"
```

Comment:
do we need to add this head file?

Reply:
We donot need it, and I have deleted this line.